src/Extension/FluentExtension.php

```diff
@@ -389,9 +389,59 @@ public function augmentSQL(SQLSelect $query, DataQuery $dataQuery = null)
             }
         }
 
+
+        // Iterate through each order clause, replacing each with the translated version
+        $order = $query->getOrderBy();
+        foreach ($order as $column => $direction) {
+            // Look for table context
+            if (preg_match('/^"(?<table>[\w\\\\]+)"\."(?<field>\w+)"$/i', $column, $matches)) {
+                $field = $matches['field'];
+            } else {
+                // If no table context is given, see if we can find one in the select
+                // e.g. "Title" -> "SomeTable"."Title"
+                $foundInConditionSearch = false;
+                foreach ($conditionSearch as $conditionSearchOption) {
+                    if (preg_match(
+                        '/^"(?<table>[\w\\\\]+)"\."(?<field>' . trim($column, '"') . ')"$/i',
```

Comment:
If $column fails the above regexp, I think assuming that it's a basic column name is a bit risky. It could be a non-trivial sub-query, or it could be a composite sort. The `else` should really be another regexp that is similar but doesn't require the table prefix.

Reply:
Ok, do you have a couple of examples that I could add to the tests?

Reply:
like `$list->sort('"Title", "Content"')`

Reply:
Sweet. We can't really make subqueries fluent in this case though right...?

Reply:
Right so you should skip them.

Reply:
I've added a bunch more tests, the last commit contains some tests that I've got no idea how to write logic for to make them pass 😆 

Also I'm a bit iffy about some of the tests in the "Add more sorting tests" commit because they're relying on the DB to return natural sort orders when the sorting isn't specific enough to always return a definite order - the fixtures have common values in them (deliberately) but maybe we either need to remove the common values to ensure that there's a predictable sort order at all times.